viewing_party/party.py

```diff
@@ -1,23 +1,197 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movies = {}
+    if title != None and genre != None and rating != None:

```

Comment:
One way to shorten this could be like the following: 
```python
if title and genre and movie:
     return {
         "title": title,
         "genre": genre,
         "rating": rating
     }
     
return None
```